app/pages/sections/additional-services/order-item/controller.js

```diff
@@ -181,7 +99,7 @@ export const saveSolutionOrderItem = async ({
       logger.info(`Follow validation errors returned from the API ${JSON.stringify(err.response.data.errors)}`);
```

Comment:
Not part of your change but should that say 'Follow ...'? I am thinking 'Following... ' but could be wrong.

Reply:
Got rid of the follow now reads as  `Validation errors ...`